### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by akintolga**
[SDAN-221]

### CODE REVIEW

File: assets/config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default {
+    displaySourceOnHome: false,
```
Comment (petrjasek) [inline assets/config.js]: is it temporary thing?

Comment (akintolga) [reply assets/config.js]: I was thinking that tomorrow another newsroom client may ask `source` to be `visible` so I created a simple config but if you think it is premature then I can get rid of it

Comment (petrjasek) [reply assets/config.js]: I'm just wondering how we would change it if it's imported from a file, maybe if you would read it from `window.newsroomConfig` or something.

Comment (akintolga) [reply assets/config.js]: Yes, that should do. After merging this PR I'll create a basic wiring for `window.newsroomConfig` to pass the configuration. Maybe we can keep this as default or something in that case.

**Pull request merged**